tests/test_wave_03.py

```diff
@@ -55,12 +55,16 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
+    assert INTRIGUE_3 in friends_unique_movies
+    assert HORROR_1 in friends_unique_movies
+    assert FANTASY_4 in friends_unique_movies
```

Comment:
Good test design by checking all the movies in `friends_unique_movies`. I could see it argued that since we're mostly concerned with the duplicated movie, `INTRIGUE_3`, in this it's only worth checking that, but I think being thorough is fine, too.